domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -132,8 +133,12 @@ class ProfileManagementController @Inject constructor(
     avatarImagePath: Uri?,
     allowDownloadAccess: Boolean,
     colorRgb: Int,
-    isAdmin: Boolean
+    isAdmin: Boolean,
+    storyTextSize: Float?,
+    appLanguage: String?,
+    audioLanguage: String?
```

Comment:
Ditto for these languages--these shouldn't be strings, they should be enums or IDs (depending on if we have a fixed list of languages or one that we need to look up at runtime). If we have a fixed list at compile-time, you could also use a proto enum here to simplify things.

Reply:
We are not sure whether it is coming from backend or we are fixing list of languages.

Reply:
App languages should be known at compile time since we technically know which translations the app is shipping with. The audio languages is trickier since we don’t know which explorations support which audio languages. Is there any API that we can use to translate ISO language code to the name of the language in Android? If so, then the easiest solution here might be to use the ISO code in the proto. That then should cover all languages even if new ones are added and the app doesn’t update.

Reply:
Added Proto for App language too. I need to check with Audio language.